src/main.cpp

```diff
@@ -28,5 +34,12 @@ struct HelloHandler : public Http::Handler {
 };
 
 int main() {
+    DatabaseAccess db = DatabaseAccess();
+    
+    string query = Fridge::genCreateQuery();
+    db.createQuery(query);
+    Fridge fridge = Fridge(1, 232);
```

Comment:
Probabil ar fi mai bine sa avem parte cu Create Table pentru frigider in fisierul script.txt. Adica tabelul sa fie deja creat inainte sa se porneasca programul ca in cazul produseror